### ISSUE TITLE
Improve PlaceCal markup for SEO & Accessibility

### ISSUE THREAD

**Issue by katjam** [geeksforsocialchange/PlaceCal#816]
- [ ] Decide canonical path for each partner
- [ ] Reflect canonical path in meta
- [ ] Add redirects
- [ ] Ad ld json
- [ ] Use audit tool to validate

**Comment by katjam** [comment]
Dream Yard. We need to do this work but it should be specified differently

**Comment by kimadactyl** [comment]
I could add in `ld+json` content for partners and events in about a day's work if you think it's worth it to get this off the starting block?

**Comment by katjam** [comment]
Maybe we make an SEO improvement milestone and prioritise within that when / if it emerges as a top-ish priority?

**Comment by kimadactyl** [comment]
Yeah and we prob need to think about what we are really trying to acheive with it past just SEO
